### DISCUSSION TITLE
when will it be fixed?

### DISCUSSION THREAD

**Discussion by jimnuts** [3kh0/ext-remover#68] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
will you guys fix it after update or no?

**Comment by 3kh0** [comment, marked as answer]
🤷🏻

**Comment by rusprice** [comment]
I'm working on creating a new exploit that will allow you to remove extensions. It's coming soon.

**Reply by phillipbruh** [reply]
how will this be used will it be a app or a link you goto if so which link is it? so i can try and see if i can come up with anything.

**Reply by Holding-THE-CREATOR** [reply]
Im making a new ingot called pingot it will bypass the latest chrome update

**Reply by Holding-THE-CREATOR** [reply]
It will take a week tho

**Reply by Holding-THE-CREATOR** [reply]
IF anyone has any bites of code that may help post them here

**Reply by ghost** [reply]
@Holding-THE-CREATOR Could you make one to add extensions?

**Reply by phillipbruh** [reply]
not possible

**Reply by ghost** [reply]
@phillipbruh  yes it is

**Reply by treetrytrey** [reply]
hey I have some javascript for that: javascript:prompt('Extension IDs here: (seperated by commas)').split(',').forEach(i=>{chrome.management.setEnabled(i.trim(),!1)})

**Comment by phillipbruh** [comment]
@Holding-THE-CREATOR 
i dunno if this is helpful but you can inject code into newly open tabs
var bypass = window.open('https://chrome.google.com/webstorex');
bypass.alert("d");

**Reply by rusprice** [reply]
Only when you run that code in chrome.google.com
